exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/sdk/trace/exporter.h"
+
+const size_t MAX_BUFFER_SIZE = 100;
```

Comment:
Let's move this inside the namespace. Otherwise we pollute the global namespace.

Reply:
Ok, done